src/pages/User/Settings/SettingsApp.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import { FC, useContext, useState } from 'react';
+import { color_gradient_glassEffect_light, font_settings } from 'style/mixins';
+import Input from 'components/Shared/Form/Input';
+import { inputNameElement } from 'helpers/mixins';
+import Submit from 'components/Shared/Form/Submit';
+import Select from 'components/Shared/Form/Select';
+import { Button } from 'components/Shared/Buttons/Button';
+import { optionsCategory } from 'data/option/category_options';
+import { optionsLanguage } from 'data/option/language_options';
+import { WrapperSettings } from 'components/Shared/containers/WrapperSettings';
+import { UserSettingsContext } from 'utils/ContextSettingsUser';
+
+export const Tittle = styled.p`
+  ${font_settings(2.4, 'normal', 600)}
+  margin: 5px auto;
+`;
+const SettingsApp = () => {
+  const {
+    defaultWordLanguage,
+    setDefaultWordLanguage,
+    defaultWordLanguageTranslate,
+    setDefaultWordLanguageTranslate,
+  } = useContext(UserSettingsContext);
+  const [selectedOptionLanguageWord, setSelectedOptionLanguageWord] = useState(defaultWordLanguage);
+  const [selectedTranslateLanguageWord, setSelectedTranslateLanguageWord] = useState(
+    defaultWordLanguageTranslate
+  );
+
```

Comment:
skoro pobierasz te wartości z contexu i masz w nim stan, a do tego tutaj sobie pobierasz funkcje (te, które napisałam, że nie wykorzystujesz) to nie ma sensu tutaj mieć stanu. 
Po prostu tam gdzie wykorzystujesz funkcje z tych useState'ów wykorzystaj te z contextu